app/mailers/user_mailer.rb

```diff
@@ -35,20 +35,22 @@ def password_reset(user, url, image, color)
     mail to: user.email, subject: t('reset_password.subtitle')
   end
 
-  def user_promoted(user, url, image, color)
+  def user_promoted(user, role, url, image, color)
     @url = url
     @admin_url = url + "admins"
     @image = image
     @color = color
-    mail to: user.email, subject: t('mailer.user.promoted.subtitle')
+    @role = role
+    mail to: user.email, subject: t('mailer.user.promoted.subtitle', role: role)
```

Comment:
Since you've set @role, you don't need to pass role as an argument. You should be able to access it directly in the email

Reply:
I'm just passing the role argument to I18n to translate the subject. I user @ROLE in the email view itself.

Reply:
Oh oops. I didn't notice that. You can ignore this comment then